.github/workflows/ci.yaml

```diff
@@ -104,7 +106,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          push: ${{ github.event_name == 'push' }}
+          push: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
```

Comment:
To allow pushing images when running manual builds

Reply:
Maybe `!= 'pull_request'`

Reply:
Done